src/Registration/pReg/Reg.py

```diff
@@ -1238,6 +1240,13 @@ def range_geometry(self):
         """Get range geometry."""
         return self.reference_image
 
+@deprecation.deprecated(details="The use of NiftyResample is discouraged and will be dropped. \
+                        Please use NiftyResampler instead.")
+
+class NiftyResample(object):
+    pass
```

Comment:
this definition will be removed by the assignment below I believe.

Reply:
That is correct, I guess, but I wrote it s.t. I could put the deprecation in front of it...because 
```
@deprecation.deprecated(details="The use of NiftyResample is discouraged and will be dropped. Please use NiftyResampler instead.")
NiftyResample = NiftyResampler
```
will generate an error.
On the other hand, none of the depreciation does anything, actually.
I can either remove both the deprecation, and the class definition, or you need to give me a hint on how to make this deprecation issue any warning :D .

Reply:
sure. it's seems tricky to get deprecate a variable. possibly cannot be done.

let's just remove the deprecation then and the class def (less code is better, especially if it's confusing!), but please add a comment that contains the word "deprecation", such that we find it when searching for it...

Otherwise, good to go from me.